lib/status.js

```diff
@@ -4,6 +4,8 @@ const WINDOW = Symbol('window');
 const BUCKETS = Symbol('buckets');
 const TIMEOUT = Symbol('timeout');
 const PERCENTILES = Symbol('percentiles');
+const BUCKET_INTERVAL = Symbol('bucket-interval');
+const EMIT_INTERVAL = Symbol('emit-interval');
```

Comment:
I think this should be `SNAPSHOT_INTERVAL`.
```suggestion
const SNAPSHOT_INTERVAL = Symbol('snapshot-interval');
```